src/plugins/list/list.h

```diff
@@ -19,6 +21,9 @@ int elektraListSet (Plugin * handle, KeySet * ks, Key * parentKey);
 int elektraListError (Plugin * handle, KeySet * ks, Key * parentKey);
 int elektraListAddPlugin (Plugin * handle, KeySet * pluginConfig);
 int elektraListEditPlugin (Plugin * handle, KeySet * pluginConfig);
+void elektraListSetIoBinding (Plugin * handle, ElektraIoInterface * binding);
```

Comment:
Is this the right place? It couples the ioplugin interface to everyone using the list plugin.

Reply:
Yes it couples it, but it is optional to call the exported function. I added more details in the comment below.

Reply:
I could not find the comment below anymore. Can you describe the deps somewhere explicitly?